sympy/calculus/codomain.py

```diff
@@ -151,3 +154,78 @@ def codomain_interval(f, set_val, *sym):
     if isinstance(domain, FiniteSet):
         return FiniteSet(*[limit(func, symbol, i) if i in FiniteSet(-oo, oo)
                             else func.subs({symbol: i}) for i in domain])
+
+
+def not_empty_in(finSet_intersection, *syms):
```

Comment:
Please don't use CamelCase, by convention they are for class names, use lowercase_with_underscores.